static/src/javascripts/projects/common/modules/experiments/tests/football-weekly-treat-vs-container.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import template from 'lodash/utilities/template';
+import treatHtml from 'raw-loader!journalism/views/footballWeeklyTreat.html';
+import containerHtml from 'raw-loader!journalism/views/footballWeeklyContainer.html';
+import config from 'lib/config';
+import { getBreakpoint } from 'lib/detect';
+import { addClassesAndTitle } from 'common/views/svg';
+
+import logo from 'svgs/journalism/football-weekly-treat/football-daily-circle480.svg';
+import button from 'svgs/journalism/football-weekly-treat/a-play-btn-black.svg';
+import waveSmall from 'svgs/journalism/football-weekly-treat/wave-small.svg';
+import waveLarge from 'svgs/journalism/football-weekly-treat/wave-large.svg';
+import waveMedium from 'svgs/journalism/football-weekly-treat/wave-medium.svg';
+
+import containerWaveLarge from 'svgs/journalism/football-weekly-container/waveform.svg';
+import containerWaveTablet from 'svgs/journalism/football-weekly-container/waveform-tablet.svg';
+import containerWaveMobile from 'svgs/journalism/football-weekly-container/waveform-mobile.svg';
+import containerWaveTiny from 'svgs/journalism/football-weekly-container/waveform-tiny.svg';
+import containerButton from 'svgs/journalism/football-weekly-container/play-btnbig.svg';
+
+const getHeadlineText = (el: Element): string =>
+    el.textContent.split(/[-–]/)[0];
+
+const runTreatTest = (): void => {
+    // Get the headline from podcast container and insert the test treat
+    const podcastContainer = document.querySelector(
```

Comment:
`document.getElementById('world-cup-daily')` (without the leading `#`) will be much faster